fms/models/__init__.py

```diff
@@ -216,6 +214,7 @@ def get_model(
     device_type: str = "cpu",
     distributed_strategy: Optional[str] = None,
     checkpoint_sharding: Optional[str] = None,
+    checkpoint_format: Optional[str] = None,
```

Comment:
This seems a little redundant if you know the source (you may be able to infer the checkpoint format based on source)

Reply:
solved with the glob rework